libretroshare/src/services/p3gxschats.cc

```diff
@@ -1133,15 +1142,15 @@ bool p3GxsChats::getGroupData(const uint32_t &token, std::vector<RsGxsChatGroup>
                 if( found != mKnownChats.end()){
                     grp.localMsgInfo = mKnownChats[RsGxsGroupId(item->meta.mGroupId)];
                 }
-//                else{
-//                    RS_STACK_MUTEX(mChatMtx);
-//                    LocalGroupInfo localMsg;
-//                    localMsg.msg ="New Join";
-//                    localMsg.update_ts = time(NULL);
-//                    localMsg.isSubscribed = true;
-//                    grp.localMsgInfo = localMsg;
-//                    mKnownChats[RsGxsGroupId(item->meta.mGroupId)] =  localMsg;
-//                }
+                else{
+                    RS_STACK_MUTEX(mChatMtx);
+                    LocalGroupInfo localMsg;
+                    localMsg.msg ="New";
+                    localMsg.update_ts = time(NULL);
```

Comment:
You better use `nullptr` instead of `NULL` otherwise new compilers will complain with a warning

Reply:
Thanks for the suggestion, we will update them in coming commits.